bodhi-server/bodhi/server/templates/update.html

```diff
@@ -826,7 +826,12 @@ <h3 class="modal-title" id="waiveModalLabel">Trigger Tests</h3>
                   waivers[waiver.testcase].push(waiver);
                 });
                 handle_unsatisfied_requirements(data);
-                handle_results(data.results, request.subject);
+                // we don't want to generate result rows for both the
+                // critpath and non-critpath decision contexts, as both
+                // queries return the same set of test results
+                if (!(request.decision_context.endsWith("critpath")) {
```

Comment:
The `endsWith()` method [does not work on Internet Explorer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith#browser_compatibility), but I don't think we really need to care about that.